.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up dependencies
+        run: |
+          sudo add-apt-repository ppa:smoser/swtpm
+          sudo apt-get update
+          sudo apt-get install qemu qemu-system-x86 qemu-utils npm swtpm
```

Comment:
Let's use the [setup script](https://github.com/RHsyseng/rhcos-slb/blob/main/tests/setup.sh), so we don't have to update this workflow every time we need to update one of our dependencies

Reply:
We will then need to maintain two setup scripts, one for `dnf`, one for `apt`. Is that still desirable?

Reply:
keeping 2 setup scripts is only slightly better than this, cause we would still need to maintain both separately. 
Let's keep it like this, then, imo it's not worth it..

Reply:
Ok ok, thanks. Could we merge this then?

Reply:
yep